packages/fixtures/src/events.ts

```diff
@@ -81,3 +86,28 @@ export const appMention = (
     ...options,
   });
 };
+
+export const memberJoinedChannel = (
+  options: Partial<MemberJoinedChannelEvent> = {},
+): EnvelopedEvent<MemberJoinedChannelEvent> => {
+  const {
+    channel: { id: channel },
+    inviter: { id: inviter },
+    team: { id: team },
+    user: { id: user },
+  } = fields;
+
+  const channel_type = options.channel
+    ? options.channel.charAt(0)
+    : channel.charAt(0);
+
+  return apiEvent<MemberJoinedChannelEvent>({
```

Comment:
```suggestion
  return apiEvent({
```

this should be covered by the function signature.